css/styles.css

```diff
@@ -34,4 +34,121 @@ li > p {
     0%{background-position:0% 50%}
     100%{background-position:200% 50%}
 }
-/*! purgecss end ignore */
\ No newline at end of file
+/*! purgecss end ignore */
+
+#sortbox:checked ~ #sortboxmenu {
```

Comment:
I know this is going to be painful to read but, can we get rid of this CSS in favor of Tailwind-style classes?

Reply:
It is an old style and it's not relevant anymore. I'll get rid of it now

Reply:
I've removed it

Reply:
I think we're good to go